js/schematic.js

```diff
@@ -276,9 +276,9 @@ navs["{{ section[0] }}"] = function(data) {
     ;
 
     // Set up the legend so it can be toggled depending on the donor.
-    tab.container(d3.select("#{{ section[0] }}"));
-
+    tabs["{{ section[0] }}"].colorScale(colorScale).grid(grid);
```

Comment:
Narrowed down the menu breakage to this commit. Branching from the commit before this.